next-types.d.ts

```diff
@@ -221,7 +223,7 @@ interface DbUserInsight {
   readonly updated_at: string;
   readonly repos: DbUserInsightRepo[];
   readonly members: DbInsightMember[];
-  readonly workspaces?: Workspace;
+  readonly workspaces?: DbUserInsightWorkspace;
```

Comment:
refactor (nit): shouldn't this be `workspace`, not `workspaces` or is that how it is in the DB already?

Reply:
It comes back from the API as `workspaces`, so I kept it the same here

Reply:
Not a high-priority @jpmcb, but I imagine migrating this property from `workspaces` to `workspace` wouldn't be a big lift?